src/pages/ContentfulPage.vue

```diff
@@ -178,22 +177,6 @@ const getContentGroups = pageData => {
 	})).filter(group => typeof group.component === 'function');
 };
 
-// Modifications for Monthly Good Landing pages
-const customMGEventsAndConfig = {
-	// Custom attribute for MG page landing specific button class
-	// TODO deprecate this when HomepageBottomCTA.vue is no longer in use
-	customCtaButtonClass: 'classic hollow',
-	// Custom attribute for event name emitted with MG landing page button clicks
-	customEventName: 'openMonthlyGoodSelector'
-};
-
-// TODO deprecate this when HomepageBottomCTA.vue is no longer in use
-const componentOptions = {
-	// Selected MG Landing page component keys to receive custom attributes
-	'homepage-bottom-cta-monthly-good': customMGEventsAndConfig,
```

Comment:
I haven't completely migrated the HomepageBottomCTA, but these 2 keys have been migrated, so we can remove the custom event code related to opening the monthly good selector, see VUE-698